torch/_inductor/scheduler.py

```diff
@@ -2736,21 +2736,12 @@ def speedup_by_fusion(
         assert device is not None
 
         def log_fusion(ms_fused: float, ms1: float, ms2: float) -> None:
-            if fusion_log.isEnabledFor(logging.DEBUG):
-                if ms_fused < ms1 + ms2:
-                    fusion_log.debug(
-                        "can fuse (benchmark): fusing %s with %s cause %sx speedup",
-                        node1.get_buffer_names(),
-                        node2.get_buffer_names(),
-                        green_text(f"{(ms1 + ms2) / ms_fused:.3f}"),
-                    )
-                else:
-                    fusion_log.debug(
-                        "cannot fuse (benchmark): fusing %s with %s cause %sx slowdown",
-                        node1.get_buffer_names(),
-                        node2.get_buffer_names(),
-                        red_text(f"{ms_fused / (ms1 + ms2):.3f}"),
-                    )
+            speedup = (ms1 + ms2) / ms_fused
+            if ms_fused < ms1 + ms2:
+                message = f"can fuse (benchmark): fusing {node1.get_buffer_names()} with {node2.get_buffer_names()} cause {green_text(f'{speedup:.3f}')}x speedup"
+            else:
+                message = f"cannot fuse (benchmark): fusing {node1.get_buffer_names()} with {node2.get_buffer_names()} cause {red_text(f'{1/speedup:.3f}')}x slowdown"
+            fusion_log.debug(message)
```

Comment:
The old implementation has the benefit that the overhead is minimal if the logging is not enabled. While the new implementation has the penalty of formatting the error message. It's not large penalty but as a convention we should avoid that.

We should at least add back `fusion_log.isEnabledFor(logging.DEBUG)`